www/templates/diary/trip_list_item.html

```diff
@@ -1,105 +1,98 @@
+                        <div class="col-90" style="font-size: 12px; text-align: center;">
+                        <b><u>{{trip.display_date}}</u></b>
+                        </div>
+                        <div class="col-10">
+                            <i class="ion-ios-more" style="font-size: 30px; color:black" ng-click="showDetail($event, trip)"></i>
+                        </div>
                     </div>
-                    <div class="col-10">
-                        <i class="ion-ios-more" style="font-size: 30px; color:black" ng-click="showDetail($event, trip)"></i>
+                    <div class="row">
+                        <div class="diary-distance-time"
+                            translate=".distance-in-time"
+                            translate-value-distance="{{ trip.display_distance }} {{trip.display_distance_suffix }}"
+                            translate-value-time="{{ trip.display_time }}">
+                        </div>
                     </div>
-                </div>
-                <div class="row">
-                    <div
-                        class="diary-distance-time"
-                        translate=".distance-in-time"
-                        translate-value-distance="{{ trip.display_distance }} {{trip.display_distance_suffix }}"
-                        translate-value-time="{{ trip.display_time }}">
+
+                    <!-- Modes / Percentages -->
+                    <div class="diary-modes-percents" ng-if="trip.percentages">
+                        <span
+                            class="diary-modes-percents-text"
+                            ng-if="!trip.isDraft"
+                            ng-repeat="sectionPct in trip.percentages">
+                            <span class="diary-modes-percents-icon {{sectionPct.icon}}"></span>
+                            <span>{{sectionPct.pct}}%</span>
+                        </span>
+                        <button class="diary-draft-button" ng-if="trip.isDraft" ng-click="explainDraft($event)">
+                            <span translate>{{'.draft'}}</span>
+                        </button>
                     </div>
-                </div>
 
-                <!-- Modes / Percentages -->
-                <div class="diary-modes-percents">
-                    <span
-                        class="diary-modes-percents-text"
-                        ng-if="!trip.isDraft"
-                        ng-repeat="sectionPct in trip.percentages"
-                    >
-                        <span class="diary-modes-percents-icon {{sectionPct.icon}}"></span>
-                        <span>{{sectionPct.pct}}%</span>
-                    </span>
-                    <button
-                    class="diary-draft-button"
-                    ng-if="trip.isDraft"
-                    ng-click="explainDraft($event)"
-                  >
-                    <span translate>{{'.draft'}}</span>
-                  </button>
-                </div>
```

Comment:
we will likely restore these when we unify the diary and label views. So we might want to leave them in there so we know where they are coming from

Reply:
These Modes / Percentages elements have not been removed - this change just fixed the formatting as the indentation of the entire file was wrong